mitmproxy/web/app.py

```diff
@@ -245,6 +245,9 @@ def put(self, flow_id):
                         request.port = int(v)
                     elif k == "headers":
                         request.headers.set_state(v)
+                    elif k == "content":
+                        print(v)
+                        response.content = str(v)
```

Comment:
I like the general idea, but this will break:

- JSON is not binary-safe, so anything binary will break this.
- JSON is super slow for multiple-MB things
- We want to have drag-and-drop upload - the easiest way to implement this is FormData upload, so we should have a multipart/formdata endpoint.

Can we put to /flow/content?